core/spec/models/spree/stock/inventory_unit_builder_spec.rb

```diff
@@ -13,14 +13,14 @@ module Stock
         it "returns an inventory unit for each quantity for the order's line items" do
           units = subject.units
           expect(units.count).to eq 3
-          expect(units.first.line_item).to eq line_item_1
-          expect(units.first.variant).to eq line_item_1.variant
+          expect(units.first.line_item_id).to eq line_item_1.id
+          expect(units.first.variant_id).to eq line_item_1.variant_id
 
-          expect(units[1].line_item).to eq line_item_2
-          expect(units[1].variant).to eq line_item_2.variant
+          expect(units[1].line_item_id).to eq line_item_2.id
```

Comment:
@priyank-gupta why are we testing it like this? Aren't testing equality against records wouldn't be better than just ids? Id can be set but record can be missing.